OpenTrons/opentrons-api
# fix(app): includeFixitCommands in GET run commands

# Overview

closes [EXEC-704](https://opentrons.atlassian.net/browse/EXEC-704).
revert do not render fixit commands and add query string to req.

## Test Plan and Hands on Testing

- load an ER protocol.
- enter ER.
- dispatch a few fixit commands.
- finish run.
- make sure there are no fixit commands in the run preview. 

## Changelog

revert commend intent logic from UI.
add `includeFixitCommands` in query string to GET run commands.

## Review requests

am I missing anyhting? 

## Risk assessment

low. no change in appearance.


[EXEC-704]: https://opentrons.atlassian.net/browse/EXEC-704?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
